pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/models/TopKBundles.java

```diff
@@ -72,6 +72,12 @@ public void update(Map<String, NamespaceBundleStats> bundleStats, int topk) {
                     pulsar.getConfiguration().isLoadBalancerSheddingBundlesWithPoliciesEnabled();
             for (var etr : bundleStats.entrySet()) {
                 String bundle = etr.getKey();
+                var stat = etr.getValue();
+
+                // skip zero traffic bundles
+                if (stat.msgThroughputIn + stat.msgThroughputOut == 0) {
```

Comment:
1.double parameter stat.msgThroughputIn  should use 
    final double EPSILON = 1e-10;
    if (Math.abs(maxBrokerBundleThroughput) < EPSILON) {
    }
2.stat.msgThroughputIn maybe <0  stat.msgThroughputIn + stat.msgThroughputIn can  = 0

Reply:
> This PR only skips zero traffic bundles.

I meant "exactly" zero traffic, when both msgThroughputIn and stat.msgThroughputIn are "exactly" zero.